README.md

```diff
@@ -24,6 +24,7 @@ Wall-E, named after the lovable character [Wall-E](https://en.wikipedia.org/wiki
 * `.iamn <arg>` - removes you from role `<arg>`
 * `.whois <arg>` - returns everyone who has role `<arg>`
 * `.roles` - displays all roles that exist on the server
+* `.here <args>` - displays all users with permissions to view the current channel. Filters results if they do not match any args, or all users if no args are given.
```

Comment:
change it to 
* `.here [<arg>]` - displays all users with permissions to view the current channel. Results can be filter by looking for users whose username or nickname on the server contains the substring indicated with any of the included `<arg>` strings or all users if no args are given. Multiple `<args>` may be entered.